subsys/mpsl/fem/Kconfig

```diff
@@ -11,6 +11,11 @@ config MPSL_FEM_NRF21540_GPIO_SUPPORT
 	bool
 	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_NORDIC_NRF21540_GPIO))
 
+config MPSL_FEM_NRF21540_GPIO_SPI_SUPPORT
+	bool
+	depends on MPSL_FEM_NRF21540_GPIO_SUPPORT
+	default $(dt_nodelabel_has_prop,nrf_radio_fem,spi-if)
```

Comment:
Please add `select EXPERIMENTAL`

Reply:
Added, however not to MPSL_FEM_NRF21540_GPIO_SPI_SUPPORT but to MPSL_FEM_NRF21540_GPIO_SPI - until now the spi-if node was present in device tree and it didn't cause any warnings, so I think it shouldn't no. 
However enabling setting `CONFIG_MPSL_FEM_NRF21540_GPIO_SPI=y` should indeed cause a warning, so added `select EXPERIMENTAL` there.